pkg/handlers/v1/subnet.go

```diff
@@ -0,0 +1,77 @@
+package v1
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+)
+
+type subnetConfiguration struct {
```

Comment:
Is it a naming convention to use camecase for defining structs in Go? Shouldn't these be camelcase with first letter upper case?

Reply:
Only needs to be uppercase if you want to export it. In this case, the struct does not need to be accessed outside of this package so lowercase is fine.